### PR TITLE
feat(siren): use app chart and upgrade version

### PR DESCRIPTION
**Pull request by mabdh**

### CODE REVIEW

**Comment by rohilsurana** [comment]
We don't need to commit the dependency app chart in the git repo, it will be fetched and packed in the CI.

File: stable/siren/values.yaml [hunk 1]
```diff
@@ -1,99 +1,122 @@
-replicaCount: 1
-podAnnotations: {}
-labels: {}
-## String to fully override siren.fullname template
-##
-# fullnameOverride:
-## String to fully override siren.name template
-##
-# nameOverride:
+siren-app:
```
Comment (rohilsurana) [inline stable/siren/values.yaml:1]: Need to add an entry for this in the `Chart.yaml -> dependencies`.

Comment (mabdh) [reply stable/siren/values.yaml:1]: Oh right, I forgot about this, thanks @rohilsurana

**Review by rohilsurana — APPROVED** [review]

**Comment by mabdh** [comment]
resolve #60

**Pull request merged**